proposals/exception-handling/Exceptions.md

```diff
@@ -274,10 +274,10 @@ try label and delegates exception handling to a `catch`/`catch_all`/`delegate`
 specified by the `try` label. For example, consider this code:
 
 ```
-try $l1
+try $l0
   try
     call $foo
-  delegate $l1  ;; (= delegate 0)
+  delegate $l0 (= delegate 0)
```

Comment:
This is a minor nit, but I liked this better with the comment `;;` (for consistency with the rest of the doc).

Reply:
Done